### PR TITLE
Removing Procedures, Fixing Errors in downloading logs

### PR DESCRIPTION
**Pull request by awholegunch**
Removes Procedures from Javadocs, fixes errors in downloading logging and obtaining list of of System Services.

Staged at: http://stg-web101.sw.joyent.continuuity.net/cdap/2.8.0-r2.8_fix_logging/en/index.html
Pages of interest:
-[v3 Service RESTful API](http://stg-web101.sw.joyent.continuuity.net/cdap/2.8.0-r2.8_fix_logging/en/reference-manual/http-restful-api/http-restful-api-v3/service.html)
-[v3 Logging RESTful API](http://stg-web101.sw.joyent.continuuity.net/cdap/2.8.0-r2.8_fix_logging/en/reference-manual/http-restful-api/http-restful-api-v3/logging.html#downloading-system-logs)

### CODE REVIEW

File: cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/logging.rst [hunk 1]
```diff
@@ -63,15 +67,24 @@ API. To do that, send an HTTP GET request::
...
+
+.. rubric:: Example
+.. list-table::
+   :widths: 20 80
+   :stub-columns: 1
+
+   * - HTTP Method
+     - ``GET <base-url>/services/system/appfabric/logs?start=1428541200&stop=1428541500``
+   * - Description
+     - Return the logs for the *AppFabric*
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/logging.rst]: the AppFabric service?

File: cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/logging.rst [hunk 2]
```diff
@@ -63,15 +67,24 @@ API. To do that, send an HTTP GET request::
 
    * - Parameter
      - Description
-   * - ``<namespace>``
-     - Namespace ID
-   * - ``<component-id>``
-     - ``appfabric``
    * - ``<service-id>``
-     - [to be completed]
+     - One of ``appfabric``, ``dataset.executor``, ``explore.service``, ``metrics``, ``streams``
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/logging.rst]: there are a couple more, right? or is this the list that the list api returns?

**Comment by nitinmotgi** [comment]
LGTM :+1: after @bdmogal comments are addressed.

File: cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/logging.rst (hunk 2 above)
Comment (awholegunch) [inline cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/logging.rst]: These are the ones with "logs: OK" in the list that the list api returns. If there are ones that should have logs that don't, that would be a bug, but I have no way of knowing which ones should and shouldn't be available. Is there a way to tell what **should** be available?

Comment (bdmogal) [inline cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/logging.rst]: @awholegunch, @gokulavasan just confirmed that we should add ``transaction`` and ``metrics.processor`` to this list.

**Pull request merged**